code/WorkInProgress/Cael_Aislinn/Rust/core_field.dm

```diff
@@ -6,7 +6,7 @@ Deuterium-tritium fusion: 4.5 x 10^7 K
 
 //#DEFINE MAX_STORED_ENERGY (held_plasma.toxins * held_plasma.toxins * SPECIFIC_HEAT_TOXIN)
 
-/obj/effect/rust_em_field
+/obj/machinery/power/rust_em_field
```

Comment:
Of course the EM field shouldn't be the fucking thing putting power in the grid then